core/block_svg.ts

```diff
@@ -1276,13 +1256,8 @@ export class BlockSvg
    */
   override removeInput(name: string, opt_quiet?: boolean): boolean {
     const removed = super.removeInput(name, opt_quiet);
-
-    if (this.rendered) {
-      this.queueRender();
-      // Removing an input will cause the block to change shape.
```

Comment:
nit: you may want to keep this comment explaining why we call bumpNeighbors (here and below, and maybe above where it wasn't before). I think it's moderately helpful... but I see you also removed it from other places so if you disagree that's fine

Reply:
In the next PR I move `bumpNeighbours` to be called as part of rendering. Whenever we rerender we should bump neighbours. So no need to have comments scattered in just some of the places.